rnn/rnn_edge.cxx

```diff
@@ -164,12 +165,21 @@ void RNN_Edge::propagate_backward(int32_t time) {
     // Log::trace("propgating backward on edge %d at time %d from node %d to node %d\n", innovation_number, time,
     // output_innovation_number, input_innovation_number);
     double delta;
-    if (output_node->node_type == MULTIPLY_NODE) {
+    if (output_node->node_type == MULTIPLY_NODE || output_node->node_type == MULTIPLY_NODE_GP) {
```

Comment:
What is the difference between multiply and multiply GP nodes?

Reply:
multiply_node has an additive bias and multiply_node_gp has a multiplicative bias.  Also, multiply_node_gp would be flagged for not training its input weights.